src/index.html

```diff
@@ -1,13 +1,509 @@
+                    Graphic Design <br>
+                    Video Production <br>
+                    Post Production <br>
+                  </p>
+
+                  <a class="card__button" href="#">
+                    <div class="card__button-box">
+                      <img
+                      src="./images/sub-title/button-arrow.svg"
+                      alt="button-arrow"
+                      class="card__button__img"
+                      >
+                    </div>
+
+                    <p class="card__button-title">learn more</p>
+                  </a>
+                </div>
+
+                <div class="card services__cards--card">
+                  <div class="card__title">003</div>
+
+                  <p class="card__description">
+                    Marketing Strategy <br>
+                    Email Marketing <br>
+                    Paid Advertising <br>
+                    Blog Content & SEO <br>
+                  </p>
+
+                  <a class="card__button" href="#">
+                    <div class="card__button-box">
+                      <img
+                      src="./images/sub-title/button-arrow.svg"
+                      alt="button-arrow"
+                      class="card__button__img"
+                      >
+                    </div>
+
+                    <p class="card__button-title">learn more</p>
+                  </a>
+                </div>
+
+                <div class="card services__cards--card">
+                  <div class="card__title">004</div>
+
+                  <p class="card__description">
+                    Digital Communications <br>
+                    Influencer Marketing <br>
+                    Product Placements <br>
+                    Strategic Partnerships <br>
+                  </p>
+
+                  <a class="card__button" href="#">
+                    <div class="card__button-box" >
+                      <img
+                      src="./images/sub-title/button-arrow.svg"
+                      alt="button-arrow"
+                      class="card__button__img"
+                      >
+                    </div>
+
+                    <p class="card__button-title">learn more</p>
+                  </a>
+                </div>
+              </article>
+            </div>
+          </div>
+        </section>
+
+        <section class="testimonials" id="testimonials">
+          <div class="conteiner">
+            <div class="testimonials__content">
+              <div class="testimonials__intro">
+                <h2 class="section-title section-title__testimonials">What <br>people say</h2>
+              </div>
+
+              <article class="testimonials__cards">
+                <div class="testimonials__card">
+                  <img
+                  src="./images/testimonials/testimonials-photo-1.png"
+                  alt="testimonials-photo-1"
+                  class="testimonials__card__img"
+                  >
```

Comment:
Need to check and fix it everywhere
```suggestion
                  <img
                    src="./images/testimonials/testimonials-photo-1.png"
                    alt="testimonials-photo-1"
                    class="testimonials__card__img"
                  >
```